Installer.php

```diff
@@ -345,12 +323,14 @@ public static function generateCookieValidationKey()
 
     protected static function generateRandomString()
     {
-        if (!extension_loaded('openssl')) {
+        if (function_exists('random_bytes')) {
+            return bin2hex(random_bytes(16));
+        }
+
+        if (!function_exists('openssl_random_pseudo_bytes')) {
             throw new \Exception('The OpenSSL PHP extension is required by Yii2.');
         }
-        $length = 32;
-        $bytes = openssl_random_pseudo_bytes($length);
-        return strtr(substr(base64_encode($bytes), 0, $length), '+/=', '_-.');
+        return bin2hex(openssl_random_pseudo_bytes(16));
```

Comment:
That doesn't seem to produce similar result.

Reply:
@samdark sure, `base64_encode()` is not the same as `base64_encode()`, but it's faster and have same format `a-zA-Z0-9`